### DISCUSSION TITLE
Ability to search Albums  via name within Mobile App

### DISCUSSION THREAD

**Discussion by JPurrier** [immich-app/immich#11835] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I extensively use albums to organise my photos. Within the web interface it is possible to place albums in list view and do a search of the album name. The mobile application should have the same feature the ability to search albums and place albums in list mode. This is very useful if you have lots of albums. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2407
